Parsers/ASimAuthentication/ProductParsers/ASimAuthenticationMicrosoftWindowsEvent.yaml

```diff
@@ -96,7 +96,7 @@ ParserQuery: |
                       TargetDomainName = tostring(EventData.TargetDomainName),
                       TargetSessionId = tostring(EventData.TargetLogonId),
                       TargetUserId = tostring(EventData.TargetUserSid),
-                      TargetUsername = tostring(iff (EventData.TargetDomainName == '-', EventData.TargetUserName, strcat(EventData.TargetDomainName, @"\" , EventData.TargetUserName))),
+                      TargetUsername = tostring(iff (EventData.TargetDomainName in ('-',''), EventData.TargetUserName, strcat(EventData.TargetDomainName, @"\" , EventData.TargetUserName))),
```

Comment:
From my test-data weirdly EventData.SubjectDomainName was always -. Didn't have any blanks. That's why I didn't bothered updating those lines.

Reply:
I think it is worth generalizing, just in case.

Reply:
okidoki. I've updated all references to this.

Technically it might be easier to to do some early-in-the-parser data cleansing/normalizing and then work from "stable" data. But not sure how this is generally approached in the ASIM parsers. E.g. we could replace all "-" with null or empty strings.